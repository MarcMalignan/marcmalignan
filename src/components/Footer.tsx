import React, { FC } from 'react';
import { styled } from '../style/theme';
import { ContentWidth } from './commons';

export const Footer: FC<{}> = () => (
  <FooterContainer>Marc Malignan Â© 2019</FooterContainer>
);

const FooterContainer = styled(ContentWidth)`
  font-size: 0.8em;
  text-align: center;
  padding: ${({ theme }) => theme.spacings.medium} 0 !important;
`;
