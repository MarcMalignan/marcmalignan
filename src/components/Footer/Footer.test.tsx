import { shallow } from 'enzyme';
import React from 'react';
import { Footer } from './Footer';

describe('Footer', () => {
  it('should match snapshot', () => {
    expect(shallow(<Footer />)).toMatchSnapshot();
  });
});
