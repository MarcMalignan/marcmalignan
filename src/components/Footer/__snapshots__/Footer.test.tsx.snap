// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<Styled(styled.div)>
  Marc Malignan Â© 2019
</Styled(styled.div)>
`;
